extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -164,41 +164,12 @@ public function handleSignupFormSubmit() {
 		$result = $response->getVal( 'result', '' );
```

Comment:
SOC-217 cleanup only. The redirect code is removed in https://github.com/Wikia/app/pull/12836.